Dockerfile

```diff
@@ -1,13 +1,4 @@
-FROM --platform=$TARGETPLATFORM ubuntu:22.04
-
-# Avoid timezone prompts during python installation
-ENV DEBIAN_FRONTEND=noninteractive
-
-RUN apt update && \
-    apt install -y libopenmpi-dev && \
-    apt install -y python3.10 && \
-    apt install -y python3-pip && \
-    pip3 install --upgrade pip
+FROM --platform=$TARGETPLATFORM python:3.10.12
```

Comment:
Maybe make the python version tag an argument here as well so you can specify the python version from the outside easily.